docs/Salesforce-batchsink.md

```diff
@@ -30,6 +30,46 @@ A Salesforce connected application must be created in order to get a client secr
 
 **SObject Name:** Salesforce object name to insert records into.
 
+There are also **sObjects** that are not supported in the Bulk API of Salesforce.
+When a job is created using an object that is not supported in the Bulk API, "_Entity is not supported by the Bulk API_" is thrown.
```

Comment:
do we always use bulk api for source and sink? Are there any cases when we use non-bulk api?

Reply:
There's one case when we don't use bulk API in source and sink. 

According to java doc in **SalesforceInputFormat.java**: 

```
Based on query length sends query to Salesforce to receive array of batch info.
If query is within limit, executes original query. If not, switches to wide object logic,
i.e. generates Id query to retrieve batch info for Ids only that will be used later
to retrieve data using SOAP API.
```

https://github.com/data-integrations/salesforce/blob/48270f0815e85f1f62b7cf73c7acf4a80a6563fe/src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceInputFormat.java#L111-L133

Reply:
I see. In that case, Could you please make sure documentation reflects those cases.

Reply:
> I see. In that case, Could you please make sure documentation reflects those cases.

PR updated accordingly.